src/griffe/agents/nodes.py

```diff
@@ -1186,12 +1187,9 @@ def get_parameter_default(node: AST, filepath: Path, lines_collection: LinesColl
     """
     if node is None:
         return None
-    if isinstance(node, NodeConstant):
-        return repr(node.value)
-    if isinstance(node, NodeStr):
-        return repr(node.s)
-    if isinstance(node, NodeName):
-        return node.id
+    value_to_str = _node_value_map.get(type(node), None)
+    if value_to_str is not None:
+        return value_to_str(node)
```

Comment:
Something simpler:

```suggestion
    with suppress(KeyError):
        return get_value(node)
```

And `from contextlib import suppress` must be added to the module imports :slightly_smiling_face:

Reply:
Good one !